.github/pull_request_template.md

```diff
@@ -1,26 +1,17 @@
-## Pull Request Info
+# Pull Request Info
 
-[PR Reviewing Guidelines](https://github.com/mongodb/docs-java/blob/master/REVIEWING.md)
-
-### Issue JIRA link:
-https://jira.mongodb.org/browse/DOCSP-NNNNN
+[JIRA](https://jira.mongodb.org)
+[Staging](https://www.example.com)
 
-### Snooty build log:
-**Paste your workerpool job link here**
+## Reviewing
 
-### Docs staging link (requires sign-in on MongoDB Corp SSO):
-https://docs-mongodborg-staging.corp.mongodb.com/java/docsworker-xlarge/NNNNNNNNNNN/
+[PR Reviewing Guidelines](https://github.com/mongodb/docs-java/blob/master/REVIEWING.md)
 
-### Self-Review Checklist
+## Self-Review Checklist
 
 - [ ] Is this free of any warnings or errors in the RST?
 - [ ] Did you run a spell-check?
-- [ ] Did you run a grammar-check?
-- [ ] Does it render on staging correctly?
+- [ ] Did you run a grammar check?
```

Comment:
Suggestion:
Both forms are technically correct, so it doesn't seem necessary to have two different forms for "spell[-]check" and "grammar[-]check".